### PR TITLE
lantiq: td-w8980: fix failsafe mode

### PR DESCRIPTION
**Pull request by ottk3**
fix failsafe-mode access for TD-W8980 (and for future builds TD-W9980) via ethernet. Kept case for readability.

Signed-off-by: Sven Rojek <git@klottenkiste.de>

### CODE REVIEW

File: target/linux/lantiq/base-files/lib/preinit/05_set_preinit_iface_lantiq [hunk 1]
```diff
@@ -10,9 +10,14 @@ set_preinit_iface() {
 	TDW8970)
 		ifname=eth0
 		;;
+	TDW8980)
```
Comment (mkresin) [inline target/linux/lantiq/base-files/lib/preinit/05_set_preinit_iface_lantiq]: Do not add a branch which set the same value as an existing branch. Use ```TDW8970|TDW8980)``` instead.

File: target/linux/lantiq/base-files/lib/preinit/05_set_preinit_iface_lantiq [hunk 2]
```diff
@@ -10,9 +10,14 @@ set_preinit_iface() {
...
+		ifname=eth0
+		;;
+	TDW9980)
```
Comment (mkresin) [inline target/linux/lantiq/base-files/lib/preinit/05_set_preinit_iface_lantiq]: Do not add code for devices which do not exists. Is very likely that a TWD9980 will never exists, since the hardware is identical to the TWD8980.

**Review by mkresin — CHANGES_REQUESTED** [review]
Please use ```git commit --amend``` to add changes to your commit. Use ```git push --force``` to overwrite your github repository/branch. The PR will afterwards update automatically.

File: target/linux/lantiq/base-files/lib/preinit/05_set_preinit_iface_lantiq [hunk 3]
```diff
@@ -10,9 +10,14 @@ set_preinit_iface() {
...
+		ifname=eth0
+		;;		
 	esac
 
 }
 
 boot_hook_add preinit_main set_preinit_iface
-
```
Comment (mkresin) [inline target/linux/lantiq/base-files/lib/preinit/05_set_preinit_iface_lantiq]: Drop this unrelated change.

**Comment by ottk3** [comment]
Thanks for your feedback. Updated the commit.
